db/migrate/20230512095053_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Let's follow the ERD on all the column names. A user should be named author in other tables. I advise you to do research about creating custom foreign keys and learn how a user will be named author in other tables.

Hint:
 ```ruby
    t.references :author, foreign_key: { to_table: 'users' } # This will create author_id and a corresponding index 
```

Reply:
Hi @elisha2kyakpo1 
- Thank you for your feedback
- I already created another migration that renames the foreign key, so you can refer to 20230512100427_change_foreign_key_for_posts.rb in the migrate folder.
-Thank you!

Reply:
@Habtamu210  you only changed the foreign key of the post table to reference author_id in this particular migration file, so you have to do same for likes and comments :+1:

Reply:
- [ ] kindly change the reference to author, you only changed the reference to author in post take a look at how your schema looks and see that you still reference user in your like and comment table, so fix this 
![Screenshot from 2023-05-12 12-33-18](https://github.com/Habtamu210/Blog-App/assets/66097371/e540950d-fe08-4202-8c09-4850d9406d5d)